ci-operator/jobs/openshift/release/openshift-release-release-4.1-periodics.yaml

```diff
@@ -1,4 +1,232 @@
 periodics:
+- agent: kubernetes
+  cron: 0 0 * * 1
+  decorate: true
+  labels:
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: longlived-aws-cluster-upgrade-4.1
+  spec:
+    containers:
+    - command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        set -euo pipefail
+
+        ci-operator --artifact-dir=$(ARTIFACTS) \
+          --secret-dir=/usr/local/pull-secret \
+          --secret-dir=/usr/local/ci-credentials \
+          --secret-dir=/usr/local/longlived-aws-cluster-credentials \
+          --template=/usr/local/longlived-test/longlived-test.yaml \
+          --input-hash=$(BUILD_ID) --input-hash=$(JOB_NAME) \
+          --target=longlived-test
+      env:
+      - name: RELEASE_IMAGE_LATEST
+        value: registry.svc.ci.openshift.org/ocp/release:4.1-ci
+      - name: CLUSTER_TYPE
+        value: aws
+      - name: CONFIG_SPEC
+        value: |
+          tag_specification:
+            name: "$(BRANCH)"
+            namespace: ocp
+          resources:
+            '*':
+              requests:
+                cpu: 10m
+          tests:
+          - as: longlived-$(CLUSTER_TYPE)-upgrade
+            commands: TEST_SUITE=all run-upgrade-tests
+            openshift_installer:
+              cluster_profile: "$(CLUSTER_TYPE)"
+              upgrade: true
+      - name: JOB_NAME_SAFE
+        value: longlived-aws-upgrade
+      image: ci-operator:latest
+      imagePullPolicy: Always
+      name: ""
+      resources:
+        requests:
+          cpu: 10m
+      volumeMounts:
+      - mountPath: /usr/local/longlived-test
+        name: job-definition
+        subPath: longlived-test.yaml
+      - mountPath: /usr/local/ci-credentials
+        name: ci-credentials
+      - mountPath: /usr/local/pull-secret
+        name: pull-secret
+      - mountPath: /usr/local/longlived-aws-cluster-credentials
+        name: longlived-aws-cluster-credentials
+    serviceAccountName: ci-operator
+    volumes:
+    - configMap:
+        name: prow-job-longlived-test
+      name: job-definition
+    - name: ci-credentials
+      secret:
+        secretName: cluster-secrets-aws
+    - name: pull-secret
+      secret:
+        secretName: ci-pull-credentials
+    - name: longlived-aws-cluster-credentials
+      secret:
+        secretName: longlived-aws-cluster-credentials
+- agent: kubernetes
+  cron: 0 12 * * 1-5
+  decorate: true
+  labels:
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: longlived-aws-cluster-e2e-4.1
```

Comment:
The name pattern seems weird.

`longlived-e2e-aws-4.1` is probably more correct.  And `longlived-maintenance-aws-4.1`.

I'd also prefer `endurance-` or `stability-` for the naming for all of this.  `longlived` lacks something.  `longrunning` is hard to understand.  We will have `soak` and `performance` variants eventually.

Reply:
going with endurance-<jobtype>-<clustertype>